### PR TITLE
BFD-3054: Update rds root credentials

### PR DESCRIPTION
**Pull request by keithdadkins**
**JIRA Ticket:**
[BFD-3054](https://jira.cms.gov/browse/BFD-3054)

**User Story or Bug Summary:**

Update RDS root credentials
---

### What Does This PR Do?

Note: I had initially added the rds root credentials in the mgmt module, forgetting they were required fields in the `aws_rds_cluster` resource and already defined in the base config. Sorry for the confusion.

_Updates_ the RDS root credentials in the base config for all environments.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] The data dictionary has been updated with any field mapping changes, if any were made.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Comment by keithdadkins** [comment]
Changes have been applied and show a clean plan.

**Review by brandoncruz3 — DISMISSED** [review]

File: ops/terraform/env/mgmt/base_config/values/mgmt.yaml [hunk 1]
```diff
@@ -48,6 +48,6 @@ bfd:
...
+      cpm_aws_account_arn: <<CIPHER>>AgV4uO1Gk4qR7y5fuMinS-89sH-rvubGMfsbLt9RfWIeG7YAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFtMjBSUE9TQm1jUU1DY1JPc2xZN3VVWXV6NURKUFZPa1NhbUhDQ2JZVDdWbEpzd1FucC9yckE1TThSTXF6MkhKQT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwFQxQtnLPhfLaTX9WYoiFsXAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMwc5l8NFpiTJjX-SQAgEQgDuYmLSpWTvJ0x52w1y7BlOuEZ2zneDa507e88E4hlmxGepLuD26IOJTyOshIQkEla-w_ZvACHnZVTR1HgIAABAAACovcmta-PG0AXBT4gzRKLzUsAZgZYErbVxjWn5huTN8Yn7hSvAQKW9nJPGl00lP_____wAAAAEAAAAAAAAAAAAAAAEAAAAgpzWRgfzpfo0KDAf3FeaaGYBnGrcOMR9iT1GyEASrFDYmg9TLX8fKaBg6o5NABMIZAGcwZQIwbHQR2niUeP1bkDRabQbjTi6o7HAzvTLa6V0ffFEAu9dVtjwRAL8gLfBAQIqsZlpzAjEAlQD-Cy7twHzUXAbWPm2bY0BEMz7MC4jV6ul0ozs0f8gvU59nVZ2QvmwwGnA2daHs<</CIPHER>>
```
Comment (mjburling) [inline ops/terraform/env/mgmt/base_config/values/mgmt.yaml:51]: Is this change related?

Comment (keithdadkins) [reply ops/terraform/env/mgmt/base_config/values/mgmt.yaml:51]: Oh, no they are not. I'm not sure why that value gets re-ciphered on my machine as there are no changes to the file. I diffed this branch with the trunk and there are no changes to the unciphered values. Either way I'll remove that change now.

Comment (keithdadkins) [reply ops/terraform/env/mgmt/base_config/values/mgmt.yaml:51]: My guess is trailing whitespaces that my editor automatically fixes.

**Review by brandoncruz3 — APPROVED** [review]

**Review by lsmitchell — APPROVED** [review]
